media-libs/jasper/jasper-4.0.0.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake-multilib
```

Comment:
I'm wondering, do we really need jasper to be multilib?

Reply:
Actually I was not sure what it is for. But it was in the old ebuild.. Will remove it.